sklearn/tests/test_docstring_parameters.py

```diff
@@ -327,21 +327,46 @@ def _get_all_fitted_attributes(estimator):
 @skip_if_no_numpydoc
 def test_precision_recall_f_score_docstring_consistency():
     """Check docstrings parameters of related metrics are consistent."""
+    metrics_to_check = [
+        metrics.precision_recall_fscore_support,
+        metrics.f1_score,
+        metrics.fbeta_score,
+        metrics.precision_score,
+        metrics.recall_score,
+    ]
     assert_docstring_consistency(
-        [
-            metrics.precision_recall_fscore_support,
-            metrics.f1_score,
-            metrics.fbeta_score,
-            metrics.precision_score,
-            metrics.recall_score,
-        ],
+        metrics_to_check,
         include_params=True,
-        # "average" - in `recall_score` we have an additional line: 'Weighted recall
-        # is equal to accuracy.'.
         # "zero_division" - the reason for zero division differs between f scores,
-        # precison and recall.
+        # precision and recall.
         exclude_params=["average", "zero_division"],
     )
+    description_regex = r"""This parameter is required for multiclass/multilabel targets\.
+        If ``None``, the metrics for each class are returned\. Otherwise, this
+        determines the type of averaging performed on the data:
+        ``'binary'``:
+            Only report results for the class specified by ``pos_label``\.
+            This is applicable only if targets \(``y_\{true,pred\}``\) are binary\.
+        ``'micro'``:
+            Calculate metrics globally by counting the total true positives,
+            false negatives and false positives\.
+        ``'macro'``:
+            Calculate metrics for each label, and find their unweighted
+            mean\.  This does not take label imbalance into account\.
+        ``'weighted'``:
+            Calculate metrics for each label, and find their average weighted
+            by support \(the number of true instances for each label\)\. This
+            alters 'macro' to account for label imbalance; it can result in an
+            F-score that is not between precision and recall\.[\s\w]*\.*
```

Comment:
can we have a comment here (and wherever we have a regex) on what it does?

Reply:
Thanks @adrinjalali ! Done so in [2f981d4](https://github.com/scikit-learn/scikit-learn/pull/29867/commits/2f981d4d8708aeb1c930176852368fcd1d4c6887), hopefully looks okay?